Mindful Moments/MainScreenView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  MainScreenView.swift
+//  Mindful Moments
+//
+//  Created by Cathy Pham on 10/17/23.
+//
+
+import SwiftUI
+
+struct MainScreenView: View {
+    var feeling: String
+    var time: String
+    
+    var body: some View {
+        HStack {
+            Text("Emotion: \(feeling)")
+            Spacer()
+            Text("Time: \(time) mins")
+        }
+        .padding(.all, 25)
+        
+        Button(action: /*@START_MENU_TOKEN@*/{}/*@END_MENU_TOKEN@*/, label: {
+            Text("Change selections")
+        })
+        .padding(.bottom, 30)
+        
+        Text("Recomended Meditations")
```

Comment:
Nit: maybe make "Recommended Meditations" Text in the ScrollView area, so when the user scrolls through the meditations, that gets pushed up while the Emotion/Time/Change Selections remains in view? Super minor nit and if you prefer it the way you have, that's totally fine!

Reply:
hmm I think either way works, @ejmercier what do u think?